sympy/stats/crv.py

```diff
@@ -263,13 +263,9 @@ def _moment_generating_function(self, t):
     def moment_generating_function(self, t, **kwargs):
         """ Moment generating function """
         if len(kwargs) == 0:
```

Comment:
What about `if not kwargs`? Which does the required by reference and not by value?